File: linearsearch.java
public class linearsearch{
    static int linear(int[] arr,int key,int n)
        {
            int index=-1;
            for(int i=0;i<n;i++)
            {
                if(arr[i]==key)
                {
                    index=i;
                    
                }
               
            }

            return index;

        }
    public static void main(String[] args) {

        
        int []arr={1,4,6,7,1};
        System.out.printf("no. is found at %d",linear(arr,1,5));
    }
}

File: practice/linearsearch.cpp
#include<iostream>
#include<vector>
using namespace std;
int  linearsearch(int n,int arr[],int key){
    int ans;
   for(int i=0;i<n;i++){
    if(arr[i]==key){
        return 1;
    }
   }  
   return 0;
}

int main(){
    int n;
    cout<<"enter size of array";
    cin>>n;
    int arr[n];
    cout<<"enter elements of array";
   

    for(int i=0;i<n;i++){
        cin>>arr[i];
    }
     int key;
    cin>>key;
    int  ans=linearsearch(n,arr,key);
    cout<<ans;
    
}